package.json

```diff
@@ -13,18 +13,19 @@
     "node",
     "node.js"
   ],
-  "main": "lib/index.js",
+  "main": "dist/index.js",
+  "browser": "browser-dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist",
-    "lib",
-    "types/index.d.ts"
+    "browser-dist"
   ],
-  "types": "types/index.d.ts",
   "scripts": {
     "build": "npm-run-all build:*",
-    "build:web": "rimraf dist && webpack",
-    "prebuild:types": "tslint -c .travis/tslint.json -p .travis/tsconfig.json",
-    "build:types": "cross-env-shell TS_NODE_PROJECT=.travis/tsconfig.json \"node $NODE_DEBUG_OPTION -r ts-node/register .travis/build-types.ts\"",
+    "build:types": "cross-env-shell TS_NODE_PROJECT=tsconfig.json \"node $NODE_DEBUG_OPTION -r ts-node/register .travis/build-types-v2.ts\"",
+    "postbuild:types": "eslint .",
```

Comment:
Since this is more rarely needed to be done, how about having it be generated after `npm install` is ran? (could also .gitignore the types file then)

```
    "generate:obs-types": "cross-env-shell TS_NODE_PROJECT=tsconfig.json \"node $NODE_DEBUG_OPTION -r ts-node/register .travis/build-types-v2.ts\"",
    "postinstall": "npm run generate:obs-types",
```

Reply:
That is a terrible idea since the script interacts with the GitHub API, if a user installs 2 packages the script will run 2 times making 4 requests to the GitHub API in total.

It uses the GitHub API to fetch the types for the latest release

Reply:
Ah yeah forgot `postinstall` is also ran for published packages _looks at sponsor spam_, meant it to be only ran if this repo is the root package (doing just `npm install` in here, not `npm install obs-websocket-js` in user's project), will check if `prepare` is better

Reply:
I don't think this is an issue, mainly because I would expect the same version of a package to not return different types for the same version if they are ever updated remotely

Reply:
Yeah I don't mean to run to update the obs types when user installs the package (since the user would only use the .d.ts in `dist` so even if script did run it wouldn't _do_ anything), it's for the development git clone this repo -> npm install -> got good state workflow)

Reply:
that could still cause problems if the user is rate-limited on the GitHub API because they did not set a token in their env vars